pyiron_atomistics/atomistics/structure/atoms.py

```diff
@@ -3104,11 +3104,9 @@ def pyiron_to_ase(pyiron_obj):
     positions = pyiron_obj.positions
     pbc = pyiron_obj.get_pbc()
     spins = pyiron_obj.get_initial_magnetic_moments()
-    if all(spins == np.array(None)) or sum(np.abs(spins)) == 0.0:
+    if np.linalg.norm(spins) == 0.0:
```

Comment:
@samwaseda, if this is legitimately a float check and not just catching the case where `spins` is a list of integer 0's, you probably want to wrap this in an `np.isclose` call. Since the target is zero, the relative tolerance is meaningless and you can disable it (`rtol=0`?)